### PR TITLE
Beatmaps Spotlights January 2018

### PR DESCRIPTION
**Pull request by Shadowthegod**
### Description
The images and news post of the January Beatmap Spotlights

### Status
- [ ] Finished

### CODE REVIEW

File: news/2018-02-20-beatmap-spotlights-january-2018.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/marigold.jpg)](https://osu.ppy.sh/beatmapsets/605710)
+
+[Spectator](https://osu.ppy.sh/users/702598)'s fine art for Deemo lovers! Collaborating with [Ascendance](https://osu.ppy.sh/users/2931883) and [Sinnoh](https://osu.ppy.sh/users/4236057), Spectator brings us Marigold by M2U feat. Guriri, an appealing song with a top quality map. Despite the relatively low BPM, the intensity of the song is well expressed with wigglings and bursts used right to the point. On top of the map itself, it contains a splendid storyboard adding to its excellence. Be sure to check out this masterpiece!
```
Comment (Xinnoh) [inline news/2018-02-20-beatmap-spotlights-january-2018.md]: Missing Ascendance in the banner unless we dont count people that map the low diffs

Comment (Shadowthegod) [reply news/2018-02-20-beatmap-spotlights-january-2018.md]: Nope, my brain just died. Fixed it.

File: news/2018-02-20-beatmap-spotlights-january-2018.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+With twisting and turning patterns of streams combined with engaging patterns on even the calmest of parts, this map just keeps you on your toes for its entirety. It also uses quite the unconventional style, using mostly circles instead of sliders for majority of the map, and with movement not commonly seen in most osu!catch maps. Excellent job by Kyuare for his rendition of this highly-popular song!
+
+## osu!mania
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/fuego.jpg)](https://osu.ppy.sh/beatmapsets/622946)
+
+Topping the osu!mania Spotlights this month we have [Couil](https://osu.ppy.sh/users/6872025) making his debut in the ranked section with Fuego! An amazing set overall with very aggressive SV usage on both of the hardest difficulties and likely one of the best implementations of such a gimmick that osu!mania has ever seen. No one should be surprised with this chart taking the cake as the HD difficulty was a very popular pick back in the MWC 4K 2017 Round of 16.
```
Comment (Xinnoh) [inline news/2018-02-20-beatmap-spotlights-january-2018.md]: lowercase fuego

File: news/2018-02-20-beatmap-spotlights-january-2018.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+If you're looking for a true challenge, look no further than the Observer difficulty. Here the idea of “comfortable" is completely shattered with intricate patterns galore that you'll see nowhere else in the Ranked section; this difficulty will certainly test the limits of your technical abilities as well as a sense of exhilaration that's hard to come by elsewhere. Keep your eyes peeled: here's a name to keep looking out for in the future!
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/perthed-again.jpg)](https://osu.ppy.sh/beatmapsets/569392)
+
+Another debut in both Ranked and Spotlights we have [TheToaphster](https://osu.ppy.sh/users/7616811) with his map of perthed again.Once again another SV fest during the whole map with a heavy and characteristic stutter usage that will make reading close to impossible, forcing you to rely on your ears and aural capabilities to get a strong performance on it.
```
Comment (Xinnoh) [inline news/2018-02-20-beatmap-spotlights-january-2018.md]: space before once again

File: news/2018-02-20-beatmap-spotlights-january-2018.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+## osu!
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/yakumo.jpg)](https://osu.ppy.sh/beatmapsets/688877)
+
+A track both known and feared by osu! veterans, [Muya](https://osu.ppy.sh/users/153323) starts us off in 2018 with a remastered set of JOINT STRUGGLE. Playing to the mapper's strengths and serving as a tribute to DJPop's version, the highest difficulty taxes player stamina with continuous sets of irregular rhythms. For those looking for a pure rhythm challenge at any difficulty level, this set will not disappoint!
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/malformed-box.jpg)](https://osu.ppy.sh/beatmapsets/627671)
+
+[Vell](https://osu.ppy.sh/users/3414541) brings us "MALFORMED BOX / INSIDE BEAST" by the Japanese band "the GazettE". The mapper manages to organise his patterns in a unique yet fitting way and represents the song's intensity by great utilization of Slider Velocity changes. Combined with an outstanding storyboard by [Darky1](https://osu.ppy.sh/users/3372928) reminiscent of the GazettE's stage performances, this is an experience you should not miss out on this month.
```
Comment (Xinnoh) [inline news/2018-02-20-beatmap-spotlights-january-2018.md]: utilization -> utilisation bc british standard

File: news/2018-02-20-beatmap-spotlights-january-2018.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+If you're looking for more unconventional beatmaps, wowaka - Unknown Mother-Goose presented by [tutuhaha](https://osu.ppy.sh/users/546991) is definitely one mapset that you shouldn't miss out on! With the song split into a fast and a slow section, tutuhaha has clearly done a wonderful job in presenting both sections by using distinctive slidershapes, coupled with a good mix of linear and circular flow that is further complemented with irregular patterns. This map also features a guest difficulty from [Habiholic](https://osu.ppy.sh/users/1974436) which is worth playing as well!
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/light-it-up.jpg)](https://osu.ppy.sh/beatmapsets/650738)
+
+Debuting this month with their first ever ranked map, [Opsi](https://osu.ppy.sh/users/8305062) - a mentee of both ProfessionalBox and Sing - shows that he can live up to high expectations with his rendition of Camellia - Light It Up. This map contains everything you know and love in a modern electronic map with its usage of complex rhythms, high paced slider velocities, and challenging streams. Definitely a sign of great things to come from this mapper.
```
Comment (Xinnoh) [inline news/2018-02-20-beatmap-spotlights-january-2018.md]: usage -> use, grammar

File: news/2018-02-20-beatmap-spotlights-january-2018.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/reality-distortion.jpg)](https://osu.ppy.sh/beatmapsets/582089)
+
+One of the most highly anticipated maps of 20..18? Although falling short of a 2017 rank, [rrtyui](https://osu.ppy.sh/users/352328)'s latest collab with [Sing](https://osu.ppy.sh/users/3795679) still managed to make rounds through the osu! community. Reality Distortion brings you modern mapping of the highest echelon with explosive slider velocity usage and fancy streams left and right. An absolute treat of a map to play, and to watch as well! The storyboard, also designed by rrtyui, features breathtakingly complex visualizations that accompany the map and song flawlessly. Even if it is out of your skill level, give the replays a watch or two with the storyboard on!
```
Comment (Xinnoh) [inline news/2018-02-20-beatmap-spotlights-january-2018.md]: visualizations -> visualisations

File: news/2018-02-20-beatmap-spotlights-january-2018.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/sengoku.jpg)](https://osu.ppy.sh/beatmapsets/564329)
+
+YUC'e - Sengoku HOP, brought to us by [Nathan](https://osu.ppy.sh/users/4785223), [Gamu](https://osu.ppy.sh/users/611174) and [appleeaterx](https://osu.ppy.sh/users/2407160), clearly shows how cuteness and glitch hop music is done, with their breathtakingly wonderful works! This map features rather gimmicky and technical patterns, including stacks and high speed sliders. Try it out if you want to challenge your reading skills and all other abilities!
```
Comment (Xinnoh) [inline news/2018-02-20-beatmap-spotlights-january-2018.md]: high speed -> high-speed

File: news/2018-02-20-beatmap-spotlights-january-2018.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+
+## osu!taiko
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/come-and-get-it.jpg)](https://osu.ppy.sh/beatmapsets/417377)
+
+It doesn't require much knowledge or time to figure out that [hikikochan](https://osu.ppy.sh/users/6512678)'s mapset of the song "Come And Get It by Krewella" follows an older osu!taiko mapping meta, a time when dubstep portions of a song were mapped without the usage of 1/8 doublets as is the common case today. This Inner Oni is outstanding by being simple, but straightforward in its patterns, as more often than not the used color (red/blue) only changes after two notes, except in rare occasions to highlight important sounds in the song.
```
Comment (Xinnoh) [inline news/2018-02-20-beatmap-spotlights-january-2018.md]: color colour

File: news/2018-02-20-beatmap-spotlights-january-2018.md [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/uta.jpg)](https://osu.ppy.sh/s/457332)
+
+You've had to have lived under a rock (underneath another rock) to not heard of this one - it's Uta mapped by [Kyuare](https://osu.ppy.sh/users/2705312)! With over 150 favorites over the course of two years, this map is quite the hyped map.
+
+With twisting and turning patterns of streams combined with engaging patterns on even the calmest of parts, this map just keeps you on your toes for its entirety. It also uses quite the unconventional style, using mostly circles instead of sliders for majority of the map, and with movement not commonly seen in most osu!catch maps. Excellent job by Kyuare for his rendition of this highly-popular song!
```
Comment (Xinnoh) [inline news/2018-02-20-beatmap-spotlights-january-2018.md]: "for majority" should be changed

File: news/2018-02-20-beatmap-spotlights-january-2018.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+
+[![](/wiki/shared/news/2018-02-20-beatmap-spotlights-january-2018/perthed-again.jpg)](https://osu.ppy.sh/beatmapsets/569392)
+
+Another debut in both Ranked and Spotlights we have [TheToaphster](https://osu.ppy.sh/users/7616811) with his map of perthed again.Once again another SV fest during the whole map with a heavy and characteristic stutter usage that will make reading close to impossible, forcing you to rely on your ears and aural capabilities to get a strong performance on it.
+
+As it looks to be a trend, SV sets seem to have very amazing lower difficulties too and this one is no exception, with the Advanced difficulty shining particularly as a very friendly and fun challenge for those who aren't quite ready to  try the heavy inverse LN stutters in the hardest difficulties.
```
Comment (Xinnoh) [inline news/2018-02-20-beatmap-spotlights-january-2018.md]: there's a double space in "to try"

File: news/2018-02-20-beatmap-spotlights-january-2018.md [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+
+Taking my time to congratulate you too Toaph and I am personally looking forward for more stuff from you!
```
Comment (Xinnoh) [inline news/2018-02-20-beatmap-spotlights-january-2018.md]: forward to more

**Comment by Shadowthegod** [comment]
Fixed them and also updated the dates.

**Pull request merged**